lua/wikis/commons/Widget/Infobox/UpcomingTournaments.lua

```diff
@@ -60,11 +61,20 @@ end
 ---@return Widget|Widget[]
 function UpcomingTournaments:_getTournaments()
 	local conditions = ConditionTree(BooleanOperator.all)
-		:add(ConditionNode(ColumnName('opponentname'), Comparator.eq, self.props.name))
 		:add(ConditionNode(ColumnName('opponenttype'), Comparator.eq, self.props.type))
 		:add(ConditionNode(ColumnName('date'), Comparator.gt, DateExt.getCurrentTimestamp() - 86400))
 		:add(ConditionNode(ColumnName('placement'), Comparator.eq, ''))
 
+	if Logic.isEmpty(self.props.name2) then
+		conditions:add(ConditionNode(ColumnName('opponentname'), Comparator.eq, self.props.name))
+	else
+		conditions:add(
+			ConditionTree(BooleanOperator.any)
+				:add(ConditionNode(ColumnName('opponentname'), Comparator.eq, self.props.name))
+				:add(ConditionNode(ColumnName('opponentname'), Comparator.eq, self.props.name2))
+		)
+	end
+
```

Comment:
Don't really like this, IMO either just call it twice, or add support to query by player instead